Prometheus/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@perses-dev/prometheus",
+  "private": true,
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+    "build": "rsbuild build",
+    "lint": "eslint src --ext .ts,.tsx"
+  },
+  "dependencies": {
+    "@module-federation/enhanced": "^0.1.11",
+    "@prometheus-io/codemirror-promql": "^0.43.0",
+    "color-hash": "^2.0.2"
+  },
+  "peerDependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@hookform/resolvers": "^3.2.0",
+    "@perses-dev/components": "0.47.0",
+    "@perses-dev/core": "0.47.0",
+    "@perses-dev/plugin-system": "0.47.0",
+    "date-fns": "^2.29.3",
+    "date-fns-tz": "^1.3.8",
+    "echarts": "5.5.0",
+    "lodash": "^4.17.21",
+    "react": "^17.0.2 || ^18.0.0",
+    "react-dom": "^17.0.2 || ^18.0.0",
+    "use-resize-observer": "^9.0.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "perses": {
+    "plugins": [
+      {
+        "display": {
+          "name": "Prometheus Datasource"
+        },
+        "kind": "PrometheusDatasource",
+        "pluginType": "Datasource"
+      },
+      {
+        "display": {
+          "name": "Prometheus Time Series Query"
+        },
+        "kind": "PrometheusTimeSeriesQuery",
+        "pluginType": "TimeSeriesQuery"
```

Comment:
`TimeSeriesQuery` or simply `Query` ? On the backend side, we don't make distinction about the different type of query for the moment.

Do you need this distinction in the frontend @jgbernalp ?

Reply:
Yes, I think we should discuss about making that distinction on the backend as well or add these base interfaces as a core plugin. This will serve as common interfaces between datasource data and charts. For example in the frontend we currently have TimeSeriesQuery and [TracesQuery](https://github.com/perses/perses/blob/main/ui/plugin-system/src/model/plugins.ts#L63).

Reply:
yeah ok, let's discuss about it in a call sometimes

Reply:
Linking back the discussion around this topic https://github.com/perses/perses/discussions/1922, but y we probably need to think about all the potential implications that "hacking" the plugin kind (to put instead what we could called a plugin "specialized" kind) would have, before going that way.

Reply:
I think in this particular case the problem is the frontend definition of the PluginModule. In reality kind will be TimeSeriesQuery and pluginType will be the actual name of the plugin. Following the kind,  metadata, spec format.

```json
{
  "kind": "PluginModule",
  "metadata": {
    "name": "BarChart"
  },
  "spec": {
    "plugins": [
      {
        "kind": "Panel",
        "metadata": {
          "name": "BarChart"
        },
        "spec": {
          "display": {
            "name": "Bar Chart"
          }
        }
      }
    ]
  }
}
```